vector/src/main/java/im/vector/riotx/features/signout/soft/epoxy/LoginErrorWithRetryItem.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.riotx.features.signout.soft.epoxy
+
+import android.widget.Button
+import android.widget.TextView
+import com.airbnb.epoxy.EpoxyAttribute
+import com.airbnb.epoxy.EpoxyModelClass
+import im.vector.riotx.R
+import im.vector.riotx.core.epoxy.VectorEpoxyHolder
+import im.vector.riotx.core.epoxy.VectorEpoxyModel
+
+@EpoxyModelClass(layout = R.layout.item_login_error_retry)
+abstract class LoginErrorWithRetryItem : VectorEpoxyModel<LoginErrorWithRetryItem.Holder>() {
+
+    @EpoxyAttribute
+    var text: String? = null
+
+    @EpoxyAttribute
+    var listener: (() -> Unit)? = null
+
+    override fun bind(holder: Holder) {
+        holder.textView.text = text
+        holder.buttonView.setOnClickListener { listener?.invoke() }
```

Comment:
create clickListener in ViewHolder (and bind the listener in bind())?

Reply:
we should do this in another optimization PR.
Also this is a copy paste of ErrorWithRetryItem class